tsconfig.node.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    /* Basic Options */
+    "target": "es2015",
+    "module": "commonjs",
```

Comment:
This produces a CommonJS version of webtreemap (i.e. `require` instead of `import`) which can be run using `node`.